package.json

```diff
@@ -103,14 +103,13 @@
     "ajv-keywords": "5.1.0",
     "base64url": "3.0.1",
     "blipp": "4.0.2",
-    "commander": "13.0.0",
+    "commander": "13.1.0",
     "cron": "3.5.0",
     "decimal.js": "10.4.3",
     "docdash": "2.0.2",
     "event-stream": "4.0.1",
     "five-bells-condition": "5.0.1",
     "glob": "10.4.3",
-    "hapi-auth-basic": "5.0.0",
```

Comment:
do we need to replace it with something like @hapi/basic or is it not needed anymore?